src/plugins/discover/public/application/components/doc/doc.tsx

```diff
@@ -22,7 +22,8 @@ import { EuiCallOut, EuiLink, EuiLoadingSpinner, EuiPageContent } from '@elastic
 import { IndexPatternsContract } from 'src/plugins/data/public';
 import { ElasticRequestState, useEsDocSearch } from './use_es_doc_search';
 import { getServices } from '../../../kibana_services';
-import { ElasticSearchHit } from '../../../../../../../../plugins/discover/public';
+import { DocViewer } from '../doc_viewer/doc_viewer';
```

Comment:
It's fine within the discover plugin. We just had it temporarily on the contract because part of discover moved to the new platform before the rest. Now it's just an implementation detail and can be used within the plugin.

We have a separate issue to pass down services nicely instead of relying on module state: https://github.com/elastic/kibana/issues/63596